qcodes/dataset/data_set.py

```diff
@@ -284,6 +287,18 @@ def __init__(self, path_to_db: str = None,
                 self._interdeps = InterDependencies_()
             self._metadata = get_metadata_from_run_id(self.conn, self.run_id)
 
+    @staticmethod
```

Comment:
@astafan8 This static method is duplicated in the experiment class. I would like to extract it into a helper function but I am not sure where that belongs in the new structure.

Reply:
it seems to belong to `qcodes.dataset.sqlite.database` as an convenient thing on top of `connect`/`get_DB_location`/etc. Is this reasonable?